### ISSUE TITLE
Type definitions

### ISSUE THREAD

**Issue by wilhelmklopp** [octokit/octokit.js#646]
👋 
I'm using Typescript for a project at the moment that also makes use of `node-github`. When compiling, the typescript compiler spits out the following warning:

```
node_modules/github/lib/index.d.ts(1463,24): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).

1463       team_reviewers?: Array;
                            ~~~~~


node_modules/github/lib/index.d.ts(1471,24): error TS2314: Generic type 'Array<T>' requires 1 type argument(s).

1471       team_reviewers?: Array;
```
node-github version: 9.3.1

**Comment by gr2m** [comment]
I don’t use TypeScript myself, can you tell me what your compiling setup is?
What node-github version are you using?

I just found out that there was a bug in the script that generates TypeScript/Flow definitions, I released a new version (12.0.2) with a fix. The definitions are here: https://unpkg.com/github@12.0.2/lib/index.d.ts I don’t see a `Array<T>` in there?

**Comment by wilhelmklopp** [comment]
Thanks for looking into this @gr2m 💖 
I think the problem was our old version of node-github, so closing this out 😄
